sspl_test/run_qa_test.sh

```diff
@@ -313,8 +337,9 @@ fi
 
 if [ "$IS_VIRTUAL" == "true" ]
 then
-    echo "Stoping the SSPL service"
-    $sudo systemctl stop sspl-ll
+    # Start mock API server if virtual machine
+    echo "Starting mock server on 127.0.0.1:$MOCK_SERVER_PORT"
+    $script_dir/mock_server &
```

Comment:
Here we are pre-ponding the stop, so it should be affected, but please verify the coverage functionality once.

Reply:
verified, haven't seen any issue.